_includes/tutorials/joining-stream-table/kstreams/markup/answer/short-answer.adoc

```diff
@@ -0,0 +1,10 @@
+Use the `builder.table()` method to create a `KTable`.
+Then use the `https://kafka.apache.org/{{ site.ak_javadoc_version }}/javadoc/org/apache/kafka/streams/kstream/ValueJoiner.html[ValueJoiner]` interface in the Streams API to join the `KStream` and `KTable`.
+
+```
+KStream<String, Rating> ratings = ...
+KTable<String, Movie> movies = ...
+final MovieRatingJoiner joiner = new MovieRatingJoiner();
+KStream<String, RatedMovie> ratedMovie = ratings.join(movies, joiner);
+ratedMovie.to(ratedMoviesTopic, Produced.with(Serdes.String(), ratedMovieAvroSerde(allProps)));
```

Comment:
Do we need this last line?  That's just printing to a Kafka topic...tangential, maybe delete it?

Reply:
I guess if people need more info for the short answer they can always go look in the code. Applied your suggestion in commit https://github.com/confluentinc/kafka-tutorials/pull/827/commits/46398d22426f8035d842b4ed661a5f894e129d22